git-log: show branch graph (#2587)
tldr-pages/tldr: pages/common/git-log.md

- Show only the first line of each commits:

`git log --oneline`

> - Show branch graph:

Reviewer: How about "Show the graph of commits in the current branch"  ?

Author: Thanks for your suggestion.

`git log --oneline`

> - Show the graph of commits in the current branch:

Reviewer 2: Perhaps this will read better?

```suggestion
- Show a graph of commits in the current branch:
```

Reviewer 3: It seems I don't have permission to push to this repo :(

Reviewer 2: @mfrw: I don't think the OP has given us permission to write the the branch of this PR.

`git log --stat`

- Show a graph of commits in the current branch:

`git log --graph`

> - Show all commits, tags and branches for the entire repo in a graph format:

Reviewer 4: For consistency with the newly introduced example:

```suggestion
- Show a graph of all commits, tags and branches in the entire repo:
```
